### PR TITLE
apache spark dashboard group exports

### PR DESCRIPTION
**Pull request by srlu94**

### CODE REVIEW

File: CollectD/Page_Apache_Spark_Admin.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   "sf_dashboard" : "Spark Worker Process",
   "sf_description" : "",
   "sf_discoveryQuery" : "something",
```
Comment (charliesignalfx) [inline CollectD/Page_Apache_Spark_Admin.json]: This needs to be removed too

File: CollectD/Page_Apache_Spark_Admin.json [hunk 2]
```diff
@@ -0,0 +1,3835 @@
...
+  "marshallId" : 2,
+  "marshallMemberOf" : [ 1 ],
+  "sf_page" : "Apache Spark Admin",
+  "sf_type" : "Page"
+}, {
+  "marshallId" : 3,
+  "marshallMemberOf" : [ 2 ],
+  "sf_dashboard" : "Spark Worker Process",
+  "sf_description" : "",
+  "sf_discoveryQuery" : "plugin: apache_spark AND spark_process: worker",
```
Comment (keitwb) [inline CollectD/Page_Apache_Spark_Admin.json]: Get rid of space after `:`.  Even if this does work, it isn't standard syntax.

File: CollectD/Page_Apache_Spark_Admin.json [hunk 3]
```diff
@@ -0,0 +1,3835 @@
...
+    "relatedDetectors" : [ ],
+    "revisionNumber" : 9,
+    "uiHelperValue" : "##CHARTID##_9"
+  }
+}, {
+  "marshallId" : 15,
+  "marshallMemberOf" : [ 2 ],
+  "sf_dashboard" : "Spark Cluster Instance",
+  "sf_description" : "",
+  "sf_discoveryQuery" : "plugin: apache_spark",
```
Comment (keitwb) [inline CollectD/Page_Apache_Spark_Admin.json]: Same thing with space.

File: CollectD/Page_Apache_Spark_Admin.json [hunk 4]
```diff
@@ -0,0 +1,3835 @@
...
+    "currentUniqueKey" : 3,
+    "revisionNumber" : 2,
+    "uiHelperValue" : "##CHARTID##_2"
+  }
+}, {
+  "marshallId" : 24,
+  "marshallMemberOf" : [ 2 ],
+  "sf_dashboard" : "Spark Overview",
+  "sf_description" : "",
+  "sf_discoveryQuery" : "plugin: apache_spark",
```
Comment (keitwb) [inline CollectD/Page_Apache_Spark_Admin.json]: Space

File: CollectD/Page_Apache_Spark_Admin.json [hunk 5]
```diff
@@ -0,0 +1,3835 @@
...
+      "name" : "New Plot",
+      "queryItems" : [ ],
+      "seriesData" : { },
+      "transient" : true,
+      "type" : "plot",
+      "uniqueKey" : 2,
+      "yAxisIndex" : 0
+    } ],
+    "chartMode" : "list",
+
```
Comment (keitwb) [inline CollectD/Page_Apache_Spark_Admin.json]: Spaces

Comment (keitwb) [inline CollectD/Page_Apache_Spark_Admin.json]: Space

**Pull request merged**